2013/12/02 17:30:28 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'usrCrediTaxi'@'localhost' (using password: YES)' in C:\xampp\htdocs\creditaxi\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\xampp\htdocs\creditaxi\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\creditaxi\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\xampp\htdocs\creditaxi\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\creditaxi\framework\gii\generators\crud\CrudCode.php(44): CModule->__get('db')
#5 C:\xampp\htdocs\creditaxi\framework\web\CFormModel.php(39): CrudCode->init()
#6 C:\xampp\htdocs\creditaxi\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\creditaxi\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\creditaxi\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\creditaxi\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\creditaxi\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\creditaxi\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(282): CController->run('')
#13 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/crud')
#14 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/creditaxi/gii/crud
HTTP_REFERER=http://localhost/creditaxi/gii/default/index
---
2013/12/02 21:15:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "login".' in C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('login')
#1 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/creditaxi/login
---
2013/12/02 21:15:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "login".' in C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\creditaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('login')
#1 C:\xampp\htdocs\creditaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\creditaxi\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/creditaxi/login
---
2013/12/04 02:12:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "site".' in C:\xampp\htdocs\crediTaxi\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(270): CController->missingAction('site')
#1 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(282): CController->run('site')
#2 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('site/site/login')
#3 C:\xampp\htdocs\crediTaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\crediTaxi\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/crediTaxi/site/site/login
HTTP_REFERER=http://localhost/crediTaxi/site/login
---
2013/12/04 02:12:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "site".' in C:\xampp\htdocs\crediTaxi\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(270): CController->missingAction('site')
#1 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(282): CController->run('site')
#2 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('site/site/login')
#3 C:\xampp\htdocs\crediTaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\crediTaxi\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/crediTaxi/site/site/login
HTTP_REFERER=http://localhost/crediTaxi/site/contact
---
2013/12/04 02:13:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "site".' in C:\xampp\htdocs\crediTaxi\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(270): CController->missingAction('site')
#1 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(282): CController->run('site')
#2 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('site/site/login')
#3 C:\xampp\htdocs\crediTaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\crediTaxi\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/crediTaxi/site/site/login
HTTP_REFERER=http://localhost/crediTaxi/site/site/login
---
2013/12/04 02:58:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "conductorController".' in C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('conductorContro...')
#1 C:\xampp\htdocs\crediTaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\crediTaxi\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/crediTaxi/conductorController
---
2013/12/04 02:59:28 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`creditaxi`.`conductor`, CONSTRAINT `FK_conduce_Taxi` FOREIGN KEY (`placa`) REFERENCES `taxi` (`placa`)). The SQL statement executed was: INSERT INTO `conductor` (`id_conductor`, `placa`) VALUES (:yp0, :yp1).
in C:\xampp\htdocs\crediTaxi\protected\controllers\ConductorController.php (73)
in C:\xampp\htdocs\crediTaxi\index.php (13)
2013/12/04 02:59:28 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`creditaxi`.`conductor`, CONSTRAINT `FK_conduce_Taxi` FOREIGN KEY (`placa`) REFERENCES `taxi` (`placa`)). The SQL statement executed was: INSERT INTO `conductor` (`id_conductor`, `placa`) VALUES (:yp0, :yp1)' in C:\xampp\htdocs\crediTaxi\framework\db\CDbCommand.php:357
Stack trace:
#0 C:\xampp\htdocs\crediTaxi\framework\db\ar\CActiveRecord.php(1023): CDbCommand->execute()
#1 C:\xampp\htdocs\crediTaxi\framework\db\ar\CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\crediTaxi\protected\controllers\ConductorController.php(73): CActiveRecord->save()
#3 C:\xampp\htdocs\crediTaxi\framework\web\actions\CInlineAction.php(49): ConductorController->actionCreate()
#4 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\crediTaxi\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\crediTaxi\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\crediTaxi\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\crediTaxi\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\crediTaxi\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\xampp\htdocs\crediTaxi\framework\web\CWebApplication.php(141): CWebApplication->runController('conductor/creat...')
#14 C:\xampp\htdocs\crediTaxi\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\crediTaxi\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/crediTaxi/conductor/create
HTTP_REFERER=http://localhost/crediTaxi/conductor/create
---
